wifi_marauder_companion/scenes/wifi_marauder_scene_console_output.c

```diff
@@ -44,19 +44,27 @@ void wifi_marauder_console_output_handle_rx_data_cb(uint8_t* buf, size_t len, vo
     furi_assert(context);
     WifiMarauderApp* app = context;
 
+    // Always save to log if enabled
     if(app->is_writing_log) {
         app->has_saved_logs_this_session = true;
         storage_file_write(app->log_file, buf, len);
     }
 
-    // If text box store gets too big, then truncate it
+    // Clear display buffer for new command responses
+    if(app->is_command && app->selected_tx_string && 
+       (strncmp("list", app->selected_tx_string, strlen("list")) == 0 ||
+        strncmp("get", app->selected_tx_string, strlen("get")) == 0)) {
+        furi_string_reset(app->text_box_store);
+        app->text_box_store_strlen = 0;
+    }
+
```

Comment:
i honestly can't remember the code, but i've tested this with marauder firmware i'll have to have another look in a few hours

Reply:
okay i see, it does work correctly it's just a bit confusing. app->selected_tx_string only contains the initial command that was sent, when the command starts receiving data, the comparison with "list" or "get" won't match anymore because the data isn't the same as the original str.